CRM/Batch/Form/Entry.php

```diff
@@ -894,17 +885,14 @@ private function processMembership(array $params) {
    *
    * @return bool
    *   true if mail was sent successfully
-   * @throws \CRM_Core_Exception
+   * @throws \CRM_Core_Exception|\API_Exception
    *
    * @deprecated
    *   This function is shared with Batch_Entry which has limited overlap
    *   & needs rationalising.
    *
    */
-  public function emailReceipt($form, &$formValues, $membership) {
-    // retrieve 'from email id' for acknowledgement
-    $receiptFrom = $formValues['from_email_address'] ?? NULL;
-
+  protected function emailReceipt($form, &$formValues, $membership): bool {
```

Comment:
emailReceipt was copied here from another class recently so we know it's not called elsewhere